openmetadata-spec/src/main/resources/json/schema/analytics/webAnalyticEventType/customEvent.json

```
@@ -0,0 +1,43 @@
+{
+  "$id": "https://open-metadata.org/schema/analytics/customEvent.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "pageViewData",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.analytics.customEvent",
```

```suggestion
  "javaType": "org.openmetadata.schema.analytics.CustomEvent",
```
Java Class name convention: should be nouns, in mixed cases with the first letter of each internal word capitalized.